Gemfile

```diff
@@ -5,10 +5,10 @@ require File.join(Bundler::Plugin.index.load_paths("bundler-inject")[0], "bundle
 
 gem "cloudwatchlogger",    "~> 0.2.1"
 gem "manageiq-loggers",    "~> 0.4.0", ">= 0.4.2"
-gem "manageiq-messaging",  "~> 0.1.2"
+gem "manageiq-messaging",  :git => 'https://github.com/bzwei/manageiq-messaging', :branch => 'rdkafka'
```

Comment:
It's not compatible with https://github.com/RedHatInsights/topological_inventory-core/pull/209/files#diff-a58d1eb0697bdcd34fcc5a30f9391a04R21